ecl/hql/hqlexpr.ipp

```diff
@@ -399,7 +399,7 @@ protected:
 
 public:
     virtual IHqlExpression *clone(HqlExprArray &newkids);
-    virtual IAtom * queryName() const { return id->lower(); }
+    virtual IAtom * queryName() const { return id ? id->queryLower() : NULL; }
```

Comment:
I'm curious why you have changed some instances of lower(id) to the construct "id ? id->queryLower() : NULL" and others to say "lower(id)"

Would be better to use lower(id) in all cases I think.